ci/do_ci.sh

```diff
@@ -196,6 +196,7 @@ elif [[ "$1" == "cmake.c++20.stl.test" ]]; then
         -DWITH_LOGS_PREVIEW=ON \
         -DCMAKE_CXX_FLAGS="-Werror $CXXFLAGS" \
         -DWITH_ASYNC_EXPORT_PREVIEW=ON \
+        -DCMAKE_CXX_STANDARD=20 \
```

Comment:
CMAKE_CXX_STANDARD=20 is valid for cmake.c++20.stl.test, ok.

For all the builds that now uses Gtest 1.13.0, which require C++14, ci/do_ci.sh most likely needs to be updated to add CMAKE_CXX_STANDARD=14 in relevant builds.